edu.cuny.hunter.streamrefactoring.core/src/edu/cuny/hunter/streamrefactoring/core/analysis/Util.java

```diff
@@ -458,6 +485,7 @@ static int getLineNumberFromIR(IBytecodeMethod method, SSAInstruction instructio
 					// type is java.lang.Object.
 					// Find the return type of the instruction.
 					TypeInference inference = TypeInference.make(node.getIR(), false);
+					seenValues.clear();
```

Comment:
Why are you clearing it here? Comments? I was speaking of other *overloaded* methods that call this method.

Reply:
No overloaded method calls `getPossibleTypes()`. Besides the method that I created, this is the only place that `getPossibleTypes()` is called.